code/obj/machinery/clonepod.dm

```diff
@@ -330,20 +330,14 @@
 			src.reagents.trans_to(src.occupant, 1000)
 
 			// Oh boy someone is cloning themselves up an army!
-		if(cloneslave && implant_master != null)
+		if(clonehack && implant_hacker != null)
 			// No need to check near as much with a standard implant, as the cloned person is dead and is therefore enslavable upon cloning.
 			// How did this happen. Why is someone cloning you as a slave to yourself. WHO KNOWS?!
```

Comment:
Couple more missed comments

(IDK if you actually care about the comments but either way I'm going to bed instead of looking for more)

Reply:
Yeah I'd like to change the comments as well. Seems like these are the last bits, thanks!